README.md

```diff
@@ -5,7 +5,7 @@ Money Manager Ex
 [![stable]][GitHubLatest] [![unstable]][GitHubDL] [![s]][SourceForgeDL]
 [![gha]][GitHubDL] [![ghl]][GitHubLatest] [![p]][packagecloud] [![w]][website]
 [![i]][GitHubIssues] [![resolution time]][isitmaintained] [![cd]][CodeDocs]
-[![c]][contributors] [![l]](LICENSE) [![o]][OpenHub]
+[![c]][contributors] [![l]](LICENSE) [![o]][OpenHub] [![Codacy Badge](https://api.codacy.com/project/badge/Grade/4cb8bd40c2304ae88251087be6b243c9)](https://www.codacy.com/app/=moneymanagerex/moneymanagerex?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=moneymanagerex/moneymanagerex&amp;utm_campaign=Badge_Grade)
```

Comment:
Link not working
Are utm params required?
Please move URLs to the end of the file.

Reply:
We shouldn't put links to private resources in the README.

How to verify the `4cb8bd40c2304ae88251087be6b243c9` project is really mmex?